src/brevitas/graph/gptq.py

```diff
@@ -51,14 +51,16 @@ def __init__(
         self.blocksize = math.ceil(self.columns / num_blocks)
 
         # Initialize Hessian matrix and counter. We need it in float32 to compute the inverse
-        self.H = torch.zeros((self.groups, self.columns, self.columns),
-                             device='cpu',
-                             dtype=torch.float32,
-                             pin_memory=torch.cuda.is_available())
-        self.B = torch.zeros((self.groups, self.columns, self.columns),
-                             device='cpu',
-                             dtype=torch.float32,
-                             pin_memory=torch.cuda.is_available())
+        self.H = torch.zeros(
+            (self.groups, self.columns, self.columns),
+            device='cpu',
+            dtype=torch.float32,
+        )
+        self.B = torch.zeros(
+            (self.groups, self.columns, self.columns),
+            device='cpu',
+            dtype=torch.float32,
+        )
```

Comment:
Maybe I should re-introduce this but I was getting weird CUDA errors. Also, since we are storing in CPU, not sure why we need to use pin_memory.
@i-colbert maybe you can comment before we merge